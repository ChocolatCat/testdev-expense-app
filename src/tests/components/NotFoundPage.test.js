/** @jest-environment jsdom */

import React from 'react';
import { render, screen } from '@testing-library/react';
import {BrowserRouter} from 'react-router-dom';
import NotFoundPage from '../../components/NotFoundPage';

it('Should render NotFoundPage', () => {
    const {asFragment} = render(
        <BrowserRouter>
            <NotFoundPage />
        </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
});