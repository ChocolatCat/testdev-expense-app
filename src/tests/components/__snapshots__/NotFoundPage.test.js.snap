// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFoundPage 1`] = `
<DocumentFragment>
  <div>
    404! Not Found!
    <a
      href="/"
    >
      Get me out of here!
    </a>
  </div>
</DocumentFragment>
`;
